src/Constants/authorization.js

```diff
@@ -0,0 +1,117 @@
+/* 
+To Dos:
+1. Testing
+*/
+
+// Spotify application details
+const stateKey = 'spotify_auth_state';
+const CLIENT_ID = 'c73c8dd43ae64b7c82a1e3b355cda443';
+// const REDIRECT_URI = 'http://localhost:8080';
+const REDIRECT_URI = 'https://curious-salamander-cadf27.netlify.app/';
+const scope = 'playlist-modify-private playlist-modify-public user-read-private user-read-email';
+
+// Function to generate a random string
+function generateRandomString(length) {
+    let text = '';
+    let possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+
+    for (let i = 0; i < length; i++) {
+        text += possible.charAt(Math.floor(Math.random() * possible.length));
+    }
+    return text;
+};
+
+// Function to set the state in localStorage
+function setStateInStorage(stateKey) {
+    const state = generateRandomString(16);
+    localStorage.setItem(stateKey, state);
+};
+
+// Function to handle login
+const handleLogin = (e) => {    
+    e.preventDefault();
+
+    //Set the state in localStorage
+    setStateInStorage(stateKey);
```

Comment:
Small note, this method has what we call a 'side-effect', where something happens inside the method that is assumed to happen (setting the state in local storage) rather than it being explicit in a return value. 

If I were to re-write this, I'd probably get rid of 'setStateInStorage' and just have the two lines of code right here:
```
    const state = generateRandomString(16);
    localStorage.setItem(stateKey, state); 
```
then you don't have to get the state from localStorage, like you're doing on line 38

Reply:
Done